gulpfile.js

```diff
@@ -149,6 +150,24 @@ function cordovaPrepare() {
   return runCommand(`cordova prepare ${platform}`);
 }
 
+function cordovaConfigureBeta() {
+  if (platform !== 'android' || !isBeta) {
```

Comment:
Nit: I would prefer to flip this function, starting with `if (platform == 'android' && isBeta) {`

Reply:
Done in 50e904e.